index.html

```diff
@@ -1,202 +1,503 @@
-							<img
-								src="./assets/icons/global/itlct-logo.svg"
-								alt="If These Lands Could Talk Logo"
-							/>
-						</a>
 
-						<!-- Button for modal -->
-						<button class="btn-la">Land Acknowledgement</button>
-					</div>
-					<!-- Links list -->
-					<!-- need to replace with the real page links -->
-					<ul>
-						<li>
-							<a href="/index.html">Home</a>
-						</li>
-						<li>
-							<!-- Need to double check once Our Impact team is started -->
-							<a href="/impact.html">Our Impact</a>
-						</li>
-						<li>
-							<a href="/values.html">Our Values</a>
-						</li>
-						<li>
-							<a href="/events.html">Events & Programs</a>
-						</li>
-						<li>
-							<a href="/team.html">Our Team</a>
-						</li>
-						<li>
-							<a href="#">Join the Team</a>
-						</li>
-						<li>
-							<a href="/contact.html">Contact</a>
-						</li>
-						<li>
-							<a href="#">Support Us</a>
-						</li>
-					</ul>
-
-					<!-- form -->
-					<form action="" class="">
-						<h2 class="h4-styles">Stay up to date with us</h2>
-						<p>
-							Register with our newsletter to receive updates on our website,
-							events, and more!
-						</p>
-						<div class="text-input">
-							<label class="hidden" for="newsletter-input">Newsletter</label>
-							<input
-								type="email"
-								name="newsletter-input"
-								id="newsletter-input"
-								size="40"
-								placeholder="Enter your email"
-								aria-label="newsletter"
-							/>
-							<button type="submit" class="button2">Submit</button>
-						</div>
-					</form>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>If These Lands Could Talk</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+		rel="stylesheet" />
+	<link rel="stylesheet" href="./css/normalize.css" />
+	<link rel="stylesheet" href="css/global.css" />
+	<!-- linked to created home.css -->
+	<link rel="stylesheet" href="./css/home.css" />
+</head>
+
+<body>
+	<!-- header section -->
+
+	<!-- =======================
+		Hero Section
+		========================== -->
+
+	<!-- "hero" is the class for the entire section -->
+	<section class="hero">
+		<!-- image of girl with dandelion -->
+		<!-- this image is only displayed on mobile and does not use a source set like the other sections -->
+		<img
+			src="./assets/images/pages/home/mobile/hero-home-mobile-2.jpg"
+			alt="a little girl holding a flower" />
+
+		<!-- "hero-home" holds the heading, paragraph and buttons for the hero image section -->
+		<!-- global class of "wrapper" is applied -->
+		<!-- "align-text" is a home-page class to center the text -->
+		<div class="hero-home wrapper align-text">
+			<h3>Preserving Tomorrow by Building Today</h3>
```

Comment:
Can you please add the two buttons here? You can use # for the href value if you don't have the urls. 

<img width="1484" alt="Screen Shot 2024-01-22 at 12 05 01 PM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/45c88414-49a8-4183-8a60-9a23d36bc656">

Reply:
removed "hidden" class from each button and left it on "btn-container" - resolved this issue.